src/promptflow/tests/test_configs/flows/activate_condition_always_met/nodeB.py

```diff
@@ -0,0 +1,8 @@
+from promptflow import tool
+
+# The inputs section will change based on the arguments of the tool function, after you save the code
```

Comment:
We may rename the script to a more meaningful name.

Reply:
Updated.